content/docs/payment-providers/quickpay/package.yml

```diff
@@ -0,0 +1,17 @@
+id: Vendr.PaymentProviders.QuickPay
+name: QuickPay
+logo: quickpay.png
```

Comment:
I'm not seeing a quickpay.png image file in this commit? This is actually breaking the preview build at the moment so we probably need to create the image and it.

Reply:
Logos need adding to `src\assets\images\logos`

Reply:
Yeah, didn't added that yet. Will see if I can find a SVG logo.